iterations_runners/iterations_runner.c

```diff
@@ -111,29 +111,40 @@ convert_str_to_int(char *s)
     return ((int) r);
 }
 
+void
+usage() {
+    printf("usage: iterations_runner_c <benchmark> <# of iterations> "
+           "<benchmark param>\n             <debug flag> [instrumentation dir] "
+           "[key] [key pexec index]\n\n");
+    printf("Arguments in [] are for instrumentation mode only.\n");
+    exit(EXIT_FAILURE);
+}
+
 int
 main(int argc, char **argv)
 {
     char     *krun_benchmark = 0;
     int       krun_total_iters = 0, krun_param = 0, krun_iter_num = 0;
-    int       krun_debug = 0, krun_num_cores = 0, krun_core;
+    int       krun_debug = 0, krun_num_cores = 0, krun_core, krun_instrument = 0;
```

Comment:
Is there a reason behind not initialising `krun_core`?

Reply:
We can do, although it will be initialised before its first use in the loop header:
```
for (krun_core = 0; krun_core < krun_num_cores; krun_core++) {
```

Reply:
OK, it just seems weird to be the odd one out on that line.